index.html

```diff
@@ -56,9 +56,9 @@
 </h2>
 <ul class="triptych see-more clearfix">
   <li class="first-li">
-    <h3>My content isn’t code.</h3>
+    <h3>My project isn’t code.</h3>
```

Comment:
I'd probably say `content` and specify that pieces of a project can be individually licensed.